fivox/cuda/simpleLFP.cu

```diff
@@ -104,7 +130,8 @@ float simpleLFP( const float* posX, const float* posY, const float* posZ,
                           volInfo.dimensions.z;
 
     // 1D grid of 1D blocks
-    dim3 blockSize( 512, 1 );
+    unsigned int nBlocks = 512;
```

Comment:
Number of blocks? shouldn't it be number of threads per block? the name is confusing